### PR TITLE
Make daml version display installed SDK versions like da list.

### PR DESCRIPTION
**Pull request by associahedron**
Makes `daml version` command a bit more useful by displaying all the installed versions (+ the latest released version even if it's not installed), a bit like `da list`. Example of the output:

```
$ daml-head version
DAML SDK versions:
  0.0.0    (active, assistant)
  0.12.16
  0.12.17  (default, latest release)
```

### CODE REVIEW

File: daml-assistant/exe/DAML/Assistant.hs [hunk 1]
```diff
@@ -69,15 +71,56 @@ main = displayErrors $ do
...
+                        <$ guard (Just v == defaultVersionM)
+                    , "assistant"
+                        <$ guard (Just v == asstVersion)
+                    , "latest release"
+                        <$ guard (Just v == envLatestStableSdkVersion)
+                    , "not installed"
+                        <$ guard (not (isInstalled v))
+                    ]
+
+                -- | Workaround for Data.SemVer old unfixed bug (see https://github.com/brendanhay/semver/pull/6)
```
Comment (neil-da) [inline daml-assistant/exe/DAML/Assistant.hs]: Yes, run screaming from code with obvious flaws that hasn't been patched in years.

File: daml-assistant/exe/DAML/Assistant.hs [hunk 2]
```diff
@@ -69,15 +71,56 @@ main = displayErrors $ do
...
+                -- TODO: move away from Data.SemVer...
+                versionCompare v1 v2 =
+                    if v1 == v2
+                        then EQ
+                        else compare v1 v2
```
Comment (neil-da) [inline daml-assistant/exe/DAML/Assistant.hs]: Ouch. Or fork SemVer.

File: daml-assistant/exe/DAML/Assistant.hs [hunk 3]
```diff
@@ -69,15 +71,56 @@ main = displayErrors $ do
...
+
+                versions = nubSortBy versionCompare (envVersions ++ fromRight [] installedVersionsE)
+                versionTable = [ (versionToText v, versionAttrs v) | v <- versions ]
+                versionWidth = maximum (1 : map (T.length . fst) versionTable)
```
Comment (neil-da) [inline daml-assistant/exe/DAML/Assistant.hs]: Should this padding code exist somewhere else as a utility?

Comment (associahedron) [reply daml-assistant/exe/DAML/Assistant.hs]: Probably! I'll extract it out if/when it becomes useful elsewhere.

File: daml-assistant/exe/DAML/Assistant.hs (hunk 1 above)
Comment (cocreature) [reply daml-assistant/exe/DAML/Assistant.hs]: fwiw, that fix was just merged after I left a comment on the issue.

Comment (cocreature) [reply daml-assistant/exe/DAML/Assistant.hs]: And we’ve got a new release!

**Pull request merged**